src/test/java/com/puppycrawl/tools/checkstyle/internal/utils/XmlUtil.java

```diff
@@ -132,4 +132,30 @@ else if (child.hasChildNodes()) {
         return result;
     }
 
+    /**
+     * Returns the value of the "name" attribute for the given node.
+     *
+     * @param node to retrieve the name
+     * @return the node name
+     */
+    public static String getNodeName(Node node) {
```

Comment:
> getNodeName
> Returns the value of the "name" attribute 

Shouldn't this be something like `getNodeNameAttributeText`? Maybe something a little bit shorter, but I don't think the current name expressly says enough about what it is doing.

Reply:
done